lib/cib/cib_attrs.c

```diff
@@ -591,7 +591,7 @@ set_standby(cib_t * the_cib, const char *uuid, const char *scope, const char *st
 
     } else {
         scope = XML_CIB_TAG_NODES;
-        attr_id = crm_strdup_printf("standby-%.256s", uuid);
+        attr_id = crm_strdup_printf("nodes-%.256s-standby", uuid);
```

Comment:
Looking at it more closely, I don't see why set_standby() sets attr_id at all. I think it should leave it NULL. If there is more than one, find_nvpair_attr_delegate() will log a warning and the value will not be updated, but I think that's less likely to cause problems than assuming the standby id is a particular string.

Alternatively, we don't really need set_standby() (which has been unused for a long time). A call to update_attr_delegate() would be sufficient.